x-pack/plugins/maps/public/classes/sources/es_search_source/util/scaling_documenation_popover.tsx

```diff
@@ -93,6 +93,14 @@ export class ScalingDocumenationPopover extends Component<Props, State> {
               </p>
             </dd>
           </dl>
+
+          <p style={{ fontStyle: 'italic' }}>
+            <FormattedMessage
+              id="xpack.maps.scalingDocs.maxResultWindow"
+              defaultMessage="{maxResultWindow} constraint provided by 'max_result_window' index setting."
+              values={{ maxResultWindow: this.props.maxResultWindow }}
```

Comment:
We can even consider adding a link here to the docs https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html#dynamic-index-settings

This link would need to be added here then too https://github.com/elastic/kibana/blob/a66551e81b730487d8f1140891a79b0733763b73/src/core/public/doc_links/doc_links_service.ts#L348